cypress/integration/tests/login.spec.jsx

```diff
@@ -0,0 +1,28 @@
+import LoginPage from "../../pages";
+// / <reference types="Cypress" />
+
+const loginPage = new LoginPage();
+
+describe("Login", () => {
+  beforeEach(() => {
+    cy.visit("localhost:8080");
+  });
+
+  it("should render input fields", () => {
+    loginPage.emailInput.should("be.visible");
+    loginPage.passwordInput.should("be.visible");
+    loginPage.submitButton.should("be.visible");
+  });
+
+  it("should succesfully login", () => {
+    loginPage.validLogin();
```

Comment:
for tests it's better to say break this function to only be login(email, password) and then chain it with a .then with the assertion you're trying to do within the .then so in the spec we can clearly see what things we're asserting on rather than call one function only and hide the rest